File: OIS/ois_rescaling/rescaling.pas
function erase(N: longint; var S: array of longint): longint;
begin
    (* insert your code here *)
    erase := 42;
end;


const
    MAXN = 5000;
var
    N, i  : longint;
    S     : array[0..MAXN-1] of longint;
    fr, fw: text;

begin
    assign(fr, 'input.txt');
    assign(fw, 'output.txt');
    reset(fr);
    rewrite(fw);
    readln(fr, N);
    for i:=0 to N-1 do
        read(fr, S[i]);

    writeln(fw, erase(N, S));
    close(fr);
    close(fw);
end.

File: OIS/ois_rescaling/rescaling.c
#include <stdio.h>
#include <assert.h>

#define MAXN 5000

int erase(int N, int S[]) {
    // insert your code here
    return 42;
}


int S[MAXN];

int main() {
    FILE *fr, *fw;
    int N, i;

    fr = fopen("input.txt", "r");
    fw = fopen("output.txt", "w");
    assert(1 == fscanf(fr, "%d", &N));
    for(i=0; i<N; i++)
        assert(1 == fscanf(fr, "%d", &S[i]));

    fprintf(fw, "%d\n", erase(N, S));
    fclose(fr);
    fclose(fw);
    return 0;
}
